rosetta/retriever/retriever.go

```diff
@@ -178,9 +185,18 @@ func (r *Retriever) Block(id identifier.Block) (*object.Block, []identifier.Tran
 		return nil, nil, err
 	}
 
-	var transactions []*object.Transaction
+	// Get up to cfg.TransactionLimit transactions as part of the block transactions
+	// and store the others in a second slice to be returned separately from the block.
+	var blockTransactions []*object.Transaction
+	var extraTransactions []identifier.Transaction
+	var idx int
```

Comment:
I have to say, I really hate your use of `idx`; it has very little benefit over an explicit `index`, but it confuses me every time, because I don't know if it's some **id**entifier. In this case, it's not even an index, and it actually took me a second to get what this simple code does because of that. It's just a count, so why not use `count`?